### ISSUE TITLE
*-sys crates reminder

### ISSUE THREAD

**Issue by gyakovlev** [cardoe/cargo-ebuild#30]
-sys crates often try to link to system libraries, and if not found will compile own private copy.
notable examples are
libgit2-sys (uses unreleased ABI and will fail at runtime if tries to link to system library), but requires setting `LIBGIT2_SYS_USE_PKG_CONFIG=1` to force looking for it.
onig-sys, will try to link to system oniguruma if it's found, and will compile private version if not.
prcre2-sys, will try system, and will compile own static if not found.
and more.

is it possible to modify cargo-ebuild to place a reminder if -sys crate is found in the depgraph? so users should check.

tracker bug in gentoo https://bugs.gentoo.org/709568

**Comment by cardoe** [comment]
Yes. A patch for the exact wording you'd like would be preferred.
